package.json

```
@@ -25,8 +25,9 @@
     "start": "npm run build:dev",
     "build:js": "webpack --mode production",
     "build:js-dev": "webpack --mode development",
-    "build:py": "python get_version_info.py && yarn copy-lib && dash-generate-components ./src/components dash_daq -p package-info.json --r-prefix 'daq'",
-    "build": "npm run test && npm run lint && npm run build:js && npm run build:py",
+    "build:py": "python get_version_info.py && yarn copy-lib && dash-generate-components ./src/components dash_daq -p package-info.json",
+    "build:r": "dash-generate-components ./src/components dash_daq -p package-info.json --r-prefix='daq'",
+    "build": "npm run test && npm run lint && npm run build:js",
```

Leave the build command as is (normal build creates the python artifacts)

```suggestion
    "build": "npm run test && npm run lint && npm run build:js && npm run build:py",
```